src/models/webhook.ts

```diff
@@ -30,11 +30,6 @@ export default class Webhook extends ManagementModel {
   save(params: {} | SaveCallback = {}, callback?: SaveCallback) {
     return this._save(params, callback);
   }
-  toJSON() {
-    const json = super.toJSON();
-    delete json['object'];
-    return json;
-  }
```

Comment:
Is this a breaking change? Is there any concern?

Reply:
Not really. This is just now redundant with the new changes I implemented in the PR. `object` is supposed to be readonly, and thus the old workaround was to delete the object before sending it. Now, because we have readonly attributes, when the parent `toJSON()` is called by `saveRequestBody` (which is called by the SDK when constructing a payload to send to the Nylas API) it will ignore the `object` key:
```
toJSON(enforceReadOnly?: boolean) { // saveReqeustBody sends in 'true'
  const json: any = {};
  const attributes = this.attributes();
  for (const attrName in attributes) {
    if (!enforceReadOnly || !attributes[attrName].readOnly) { // 'object'.readOnly === true
      const attr = attributes[attrName];
      json[attr.jsonKey] = attr.toJSON((this as any)[attrName]);
    }
  }
  return json;
}
```